app/views/ats/positions/pipeline_tab.html.slim

```diff
@@ -132,11 +132,17 @@ ruby:
             = label_tag :assigned_only, "Assigned only", class: "form-check-label"
 
     /! MOBILE ONLY STAGE NAVIGATION
-    #pipeline-mobile-stage-nav.d-sm-none.nav.nav-pills.d-flex.justify-content-between.mt-3.mb-2 role="tablist"
-      .d-sm-none.dropdown.w-100.profile-mobile-nav data-controller="pipeline-column"
-        button.btn.form-control.d-flex.align-items-center.text-dark.dropdown-toggle.w-100.px-3 [
-            id="dropdownNavLink" data-bs-toggle="dropdown"]
-          span.hstack.gap-2 data-pipeline-column-target="dropdownPlaceholder"
+    #pipeline-mobile-stage-nav.d-sm-none.vstack.mt-3.mb-2 role="tablist"
+      .dropdown.profile-mobile-nav data-controller="pipeline-column"
+        = render ButtonComponent.new( \
+                   variant: :cancel,
+                   type: :button,
+                   id: "dropdownNavLink",
+                   class: "dropdown-toggle w-100",
+                   flex_content_position: :between,
```

Comment:
I don't see a difference when I change it to `:start`. So, is it really necessary to add `:between`? :thinking:

Reply:
Start:
![image](https://github.com/user-attachments/assets/c1f50efe-871d-412c-aed5-754370bb7ba7)

Between:
![image](https://github.com/user-attachments/assets/db15c248-0094-4903-b158-fe621e3eced2)